src/fireblocks-sdk.ts

```diff
@@ -89,6 +91,23 @@ export class FireblocksSDK {
         return await this.apiClient.issueGetRequest(url);
     }
 
+    /**
+     * Gets a list of vault accounts per page matching the given filter or path
+     * @param pagedVaultAccountsRequestFilters Filters for the first request
+     * @param nextOrPreviousPageUrl URL containing the full path for the previous/next page as returned from previous request result
+     */
+    public async getVaultAccountsWithPageInfo(pagedVaultAccountsRequestFilters?: PagedVaultAccountsRequestFilters, nextOrPreviousPageUrl?: string): Promise<PagedVaultAccountsResponse> {
+        if (nextOrPreviousPageUrl) {
+            const index = nextOrPreviousPageUrl.indexOf("/v1/");
+            const path = nextOrPreviousPageUrl.substring(index, nextOrPreviousPageUrl.length);
```

Comment:
why do u need to do this ? does the base path has no meaning ?
if so - why not returning only the path in the response rather than the full Url ?
(can the user use the full Url directlly in the console and get proper results ?)

Reply:
the users can access the developer-api directly with URLs or through the sdk.. that's why we return the full URL for next/previous page.
We don't need the base path in the sdk because the axios-client is already initiated with the developer-api base url.